components/FilterBar/FilterBar.js

```diff
@@ -1,6 +1,6 @@
 import { useRecoilState } from 'recoil'
-import InputContainer from './InputContainer'
-import { formFiltersState } from './Catalog'
+import InputContainer from '../InputContainer/InputContainer'
```

Comment:
Components should be imported from the folder root - index.js so there won't be any excess in path